src/components/EventList.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import { mount } from '@cypress/react';
+
+import EventList from './EventList';
+
+describe('EventList', () => {
+    let dummyOrg;
+    let dummyEvents;
+
+    beforeEach(()=> {
+        cy.fixture('dummyOrg.json')
+            .then((data) => {
+                dummyOrg = data;
+            });
+        cy.fixture('dummyEvents.json')
+            .then((data) => {
+                dummyEvents = data;
+            });
+    });
+
+    it('contains data for each event', () => {
+        mount(<EventList events={ dummyEvents.data } org={ dummyOrg.data }/>);
+
+        cy.get('[data-test="event-list"]>[data-test="event"]').each((item) => {
+            cy
+                .wrap(item)
+                .get('[data-test="event-title"]').should('be.visible')
+                .get('[data-test="org-title"]').should('be.visible')
+                .get('[data-test="campaign-title"]').should('be.visible')
+                .get('[data-test="start-time"]').should('be.visible')
+                .get('[data-test="end-time"]').should('be.visible')
+                .get('[data-test="location-title"]').should('be.visible');
```

Comment:
This feels off to me. I think we should really consider how we test components like this as we carry on. No need to change anything now, but lets make a mental note and try to figure out (or research) some best practices.

Reply:
Yes, I agree!